protobuf-definitions/proto/mlagents/envs/communicator_objects/unity_rl_output.proto

```diff
@@ -9,7 +9,6 @@ message UnityRLOutput {
     message ListAgentInfoProto {
         repeated AgentInfoProto value = 1;
     }
-    bool global_done = 1;
```

Comment:
Need to regenerate the associated code for this, right? This sounds like a better way to "delete" the field: https://developers.google.com/protocol-buffers/docs/proto#reserved

Reply:
Yes, but I think I should not regenerate the files at this stage so we have less diff files to look at. Also, it will make it easier to have parallel PRs on the proto definitions without messing up the generated files.

Reply:
Sounds good on not regenerating in this PR, but I would still suggest replacing this line with
```
reserved 1; // deprecated bool global_done field
```